_releaser/main.go

```diff
@@ -29,3 +30,19 @@ func main() {
 		}))
 	ctx.FatalIfErrorf(ctx.Run())
 }
+
+// getEnvOrSecret retrieves secret's value from secret file or env
+func getEnvOrSecret(name string) string {
+	if v, ok := os.LookupEnv(name); ok {
+		return v
+	}
+	secretPath := "/run/secrets/" + name
```

Comment:
I guess "in practice" this won't be an issue, but perhaps if we want to do it "correct";


```suggestion
	secretPath := filepath.Join("/run/secrets", name)
```

Reply:
Done thanks